### PR TITLE
Upgrade to hono 1.4.0

### PR DESCRIPTION
**Pull request by sophokles73**

### CODE REVIEW

File: charts/hono/templates/hono-adapter-amqp/headless-svc.yaml [hunk 1]
```diff
@@ -1,16 +0,0 @@
-{{- if and .Values.adapters.amqp.enabled .Values.prometheus.createInstance }}
```
Comment (calohmn) [inline charts/hono/templates/hono-adapter-amqp/headless-svc.yaml:1]: The headless services should then also be removed from the `hono/README.md` file.

Btw., what is the remaining `eclipse-hono-service-device-registry-headless` service used for?

Comment (sophokles73) [reply charts/hono/templates/hono-adapter-amqp/headless-svc.yaml:1]: exactly

**Comment by sophokles73** [comment]
@calohmn done ...

**Review by calohmn — APPROVED** [review]
LGTM

**Pull request merged**